packages/embed/src/initializers/initialize.ts

```diff
@@ -14,14 +14,18 @@ export const initialize = (
     includeCss(cssFilename)
   }
 
-  Array.from(embedTypeElements).forEach((element, index) => {
+  for (let index = 0; index < embedTypeElements.length; index += 1) {
+    const element = embedTypeElements.item(index)
     if (forceReload || element.dataset.tfLoaded !== 'true') {
       const formId = element.getAttribute(embedElementAttribute)
       if (!formId) {
         throw new Error(`Invalid ${embedElementAttribute}=${formId} for embed #${index}`)
       }
+      if (element.getAttribute('data-tf-hubspot') !== null) {
```

Comment:
how about `undefined`? Maybe worth changing to `!= null` to cover `undefined` as well?

Reply:
```suggestion
      if (element.getAttribute('data-tf-hubspot')) {
```
An alternative here could be checking straightforward for a [Truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) value

Reply:
When the attribute is not present the [return value is `null`](https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute#non-existing_attributes). However if it is present, the value is an empty string and therefore [Falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) :D

Such is a life with JavaScript and browser APIs 😛